### ISSUE TITLE
Util class to persist Metric Snapshot 

### ISSUE THREAD

**Issue by freddyDOTCMS** [dotCMS/core#27179]
### User Story

To persist the Metric data this Util Class need to collect if first from:

- the Static Snapshot data: this is all the date that can be taken directly from the dotCMS Data Base, in this point we already are going to have a Util Method to get all this data implemented here https://github.com/dotCMS/core/issues/27177.
- The "Dynamic" Data: This is the data  collect from the API hit. in this point we are going to have a method to get all this data implemented here https://github.com/dotCMS/core/issues/27178.

We need to merge this to type of data en merge it before sent it to persist.

After collect the Data it need to be sent to a End Point using a POST HTTP Request, this ENd point is going to be implemented in another issue.

### Acceptance Criteria

Persist the Metric SnapShot hitting the End point

### Proposed Objective

Core Features

### Proposed Priority

Priority 2 - Important

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/com.dotcms.experience/pull/21

**Comment by dsilvam** [comment]
Blocked by https://github.com/dotCMS/core/issues/27170

**Comment by dsilvam** [comment]
Passing this one. Remaining work, if any, will be included as part of https://github.com/dotCMS/core/issues/27130
